CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+- Add a header with the general information on factories usage [#99](https://github.com/palkan/test-prof/issues/99) ([@szemek][])
```

Comment:
Don't forget to link to you GitHub handle at the bottom of the changelog (right now it's not clickable)